internal/testhelpers/tagsuite/tagsuite.go

```diff
@@ -57,6 +57,7 @@ const (
 	Secrets         = "secrets"
 	Switch          = "switch"
 	Shell           = "shell"
+	FindProject     = "findproject"
```

Comment:
Is this used?

Reply:
Ah nope, thanks for spotting. I had started implementing tests but it was too much of a scope creep, so I filed https://activestatef.atlassian.net/browse/DX-1222